web/services/getUserProjectCustomerReportJsonService.php

```diff
@@ -142,53 +139,53 @@
 
             $totalHours[$projectName] = 0;
 
-            $record[project] = $projectName;
+            $record['project'] = $projectName;
 
             $record['id'] = $projectId;
 
             $totalHours[$projectName] += round($hours, 2, PHP_ROUND_HALF_DOWN);
-            $totalHours[total] += round($hours, 2, PHP_ROUND_HALF_DOWN);
+            $totalHours['total'] += round($hours, 2, PHP_ROUND_HALF_DOWN);
 
-            $record[total] = $totalHours[$projectName];
+            $record['total'] = $totalHours[$projectName];
 
             $records[] = $record;
         }
 
         foreach($records as $record)
         {
-            $record[percentage] = round(100 * $totalHours[$record[project]]/$totalHours[total], 2, PHP_ROUND_HALF_DOWN);
-            $response[records][] = $record;
+            $record['percentage'] = round(100 * $totalHours[$record['project']]/$totalHours['total'], 2, PHP_ROUND_HALF_DOWN);
+            $response['records'][] = $record;
```

Comment:
I wonder how this was working before 😅

Reply:
Hehe, PHP is extremely permissive. But it will stop working at some point, according to the error message:
```
[php7:warn] [pid 23627] PHP Warning:  Use of undefined constant sortable - assumed 'sortable' (this will throw an Error in a future version of PHP)